peachjam/adapters/gazettes.py

```diff
@@ -29,31 +29,42 @@ def check_for_updates(self, last_refreshed):
             f"Checking for new gazettes from Gazettes.Africa since {last_refreshed}"
         )
 
-        queryset = (
+        # start building queryset for updated gazettes
+        updated_qs = (
             CoreDocument.objects.filter(doc_type="gazette")
             .non_polymorphic()
             .using("gazettes_africa")
         )
+        # start building queryset for deleted gazettes
+        deleted_qs = Gazette.objects.filter(jurisdiction=self.jurisdiction)
 
         if "-" not in self.jurisdiction:
             # locality code not specified hence None e.g "ZA"
-            queryset = queryset.filter(jurisdiction=self.jurisdiction, locality=None)
+            updated_qs = updated_qs.filter(
+                jurisdiction=self.jurisdiction, locality=None
+            )
         else:
-            queryset = queryset.filter(jurisdiction=self.jurisdiction.split("-")[0])
+            updated_qs = updated_qs.filter(jurisdiction=self.jurisdiction.split("-")[0])
 
             # locality code present e.g. "ZA-gp"
             if self.jurisdiction.split("-")[1] != "*":
-                queryset = queryset.filter(
-                    locality__code=self.jurisdiction.split("-")[1]
-                )
+                locality_code = self.jurisdiction.split("-")[1]
+                updated_qs = updated_qs.filter(locality__code=locality_code)
+                deleted_qs = deleted_qs.filter(locality_code=locality_code)

```

Comment:
is the single underscore in `locality_code` correct?

Reply:
Added :heavy_check_mark: